server.js

```diff
@@ -1,8 +1,47 @@
-// Import path interpreted by the Remix compiler
+import { createRequestHandler } from "@remix-run/netlify";
 import * as build from "@remix-run/dev/server-build";
-import { createRequestHandler } from "@remix-run/netlify-edge";
-export default createRequestHandler({
+
+/*
+ * Returns a context object with at most 3 keys:
+ *  - `netlifyGraphToken`: raw authentication token to use with Netlify Graph
+ *  - `clientNetlifyGraphAccessToken`: For use with JWTs generated by
+ *    `netlify-graph-auth`.
+ *  - `netlifyGraphSignature`: a signature for subscription events. Will be
+ *    present if a secret is set.
+ */
+function getLoadContext(event, context) {
+  let rawAuthorizationString;
+  let netlifyGraphToken;
+
+  if (event.authlifyToken != null) {
+    netlifyGraphToken = event.authlifyToken;
+  }
+
+  const authHeader = event.headers["authorization"];
+  const graphSignatureHeader = event.headers["x-netlify-graph-signature"];
+
+  if (authHeader != null && /Bearer /gi.test(authHeader)) {
+    rawAuthorizationString = authHeader.split(" ")[1];
+  }
+
+  const loadContext = {
+    clientNetlifyGraphAccessToken: rawAuthorizationString,
+    netlifyGraphToken: netlifyGraphToken,
+    netlifyGraphSignature: graphSignatureHeader,
+  };
+
+  // Remove keys with undefined values
+  Object.keys(loadContext).forEach((key) => {
+    if (loadContext[key] == null) {
+      delete loadContext[key];
+    }
+  });
+
+  return loadContext;
+}
```

Comment:
This Graph stuff can be removed

Reply:
I wasn’t sure if it was still being used at all. I’ll go ahead and remove it along with the `getLoadContext`

Reply:
Existing sites might use it, but it should be removed from the template for new sites, as Graph is deprecated